youtube_dl/extractor/morningstar.py

```diff
@@ -1,13 +1,11 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
-import re
-
 from .common import InfoExtractor
+from ..utils import get_element_by_id
 
 
 class MorningstarIE(InfoExtractor):
-    IE_DESC = 'morningstar.com'
```

Comment:
Other extractors don't usually specify `IE_DESC` to include `.com`, and it isn't part of Morningstar's name.